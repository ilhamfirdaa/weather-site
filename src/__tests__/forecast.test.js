// import React from 'react'
// import { mount } from 'enzyme'
// // import renderer from 'react-test-renderer'

// import { PureForecast } from '../pages/index'
// import data from '../data'

// describe('Components testing', () => {
//   // it('shows "Weather site!"', () => {
//   //   const tree = mount(<PureForecast data={data} />)

//   //   expect(tree.find('h4').text()).toEqual('Weather site!')
//   // })

//   it('contain image', () => {
//     const tree = mount(<PureForecast data={data} />)

//     expect(tree.contains(<img alt="" />)).toBeDefined()
//   })
// })

// // describe('Snapshot testing', () => {
// //   it('matching?', () => {
// //     const tree = renderer.create(<PureForecast data={data} />).toJSON()
// //     expect(tree).toMatchSnapshot()
// //   })
// // })
