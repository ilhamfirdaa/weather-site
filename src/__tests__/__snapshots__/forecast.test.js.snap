// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing matching? 1`] = `
<main
  className="makeStyles-mainContainer-1"
>
  <div
    className="makeStyles-overlay-2"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd"
    >
      <h2
        className="MuiTypography-root MuiTypography-h2"
      >
        Weather site!
      </h2>
      <h3
        className="MuiTypography-root MuiTypography-h3"
      >
        Jakarta
      </h3>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Overcast clouds
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Light rain
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Broken clouds
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Heavy rain
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Light rain
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Broken clouds
        </p>
      </li>
      <li
        style={
          Object {
            "display": "inline",
            "listStyle": "none",
            "textAlign": "center",
          }
        }
      >
        <p>
          Heavy rain
        </p>
      </li>
    </div>
  </div>
</main>
`;
